README.md

```diff
@@ -4,6 +4,12 @@
 
 Allows one to easily build a counterparty-server, `counterblock` and/or Counterwallet system, with all required components. Uses Docker and Docker compose.
 
-See documentation at <http://counterparty.io/docs/federated_node/>.
```

Comment:
hm yeah I don't want to include the federated node docs in the official documentation... I think we should just have everything in this repo.